06_homework_online_store/script.js

```diff
@@ -4,113 +4,106 @@ let bikes = [{
     engineCapacity: 980,
     colour: "white",
     imgSrc: 'images/bmw.jpg',
-    price: 2500,
+    price: "2500 Euro",
```

Comment:
What I meant was structure like this:
```json
{
  ...
  price: 2500,
  currency: "EUR"
},
{
  ...
  price: 100000,
  currency: "RUB" 
}
```

and usage would be 
```
`${price} ${currency}`
```

Reply:
Thank you! Done.